distributions/kfdef/kfctl_openshift_v1.3.0.yaml

```diff
@@ -13,7 +13,7 @@ spec:
   - kustomizeConfig:
       repoRef:
         name: manifests
-        path: distributions/stacks/openshift/application/istio-1-9-Openshift
+        path: distributions/stacks/openshift/application/istio-1-9-0-Openshift
```

Comment:
Should we rename the dir referenced here from `istio-1-9-0-Openshift` to `istio-1-9-5-Openshift` since that is what is actually being deployed?

Reply:
@vpavlin thats for KF 1.3.1, this is specific for release 1.3.0 which is 1-9-Openshift